ui/admin/app/components/mapping-list/index.js

```diff
@@ -0,0 +1,33 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+export default class MappingListComponent extends Component {
+  @tracked
+  newOptionKey;
+  @tracked
+  newOptionValue;
+
+  @action
+  updateArray() {}
```

Comment:
do we need this?

Reply:
Woops good catch, that was a leftover action that I forgot to remove.